src/PBot/SyncOMatic/DefaultTemplateRepo.cs

```diff
@@ -31,5 +30,22 @@ static DefaultTemplateRepo()
                 Parts = new Parts("Particular/RepoStandards", TreeEntryTargetType.Blob, "master", "src/RepoName.sln.DotSettings")
             });
         }
+
+        public static List<SyncItem> WithLicense(string license)
+        {
+            if (license == null)
+            {
+                return ItemsToSync;
+            }
+
+            var items = new List<SyncItem>(ItemsToSync);
+            var licenseFile = license != "standard" ? $"{license.ToUpper()}.md" : "RPL.md";
+            items.Add(new SyncItem
+            {
+                Target = "LICENSE.md",
```

Comment:
the file should be just `LICENSE`, no extension, see https://developer.github.com/v3/licenses/